tests/stdlib/tjsontestsuite.nim

```diff
@@ -19,19 +19,19 @@ let parsing_testdata = {
   "i_string_incomplete_surrogates_escape_valid": """["\uD800\uD800\n"]""",
   "i_string_inverted_surrogates_U+1D11E": """["\uDd1e\uD834"]""",
   "i_string_lone_second_surrogate": """["\uDFAA"]""",
-  "i_string_not_in_unicode_range": """["�"]""",
-  "i_string_truncated-utf-8": """["��"]""",
```

Comment:
What's up with these?

Reply:
Sorry, fixed. Not sure what vscode did to this file.

Reply:
I didn't know GitHub cannot show my changes properly

https://github.com/nim-lang/Nim/pull/20343/commits/d93fb0c96aa73cdc14ec1585680759477d7c50db

At least this file is not changed anymore.